products/sle/main.pm

```diff
@@ -1049,6 +1056,14 @@ elsif (get_var("Y2UITEST")) {
 elsif (get_var("WINDOWS")) {
     loadtest "installation/win10_installation.pm";
 }
+elsif (ssh_key_import) {
+    loadtest "boot/boot_to_desktop.pm";
+    loadtest "x11/ssh_key_check.pm";
+    loadtest "x11/reboot_and_install.pm";
+    load_inst_tests();
+    load_reboot_tests();
+    loadtest "x11/ssh_key_verify.pm";
+}
```

Comment:
because this section is new and very special, i.e. the "reboot_and_install" part, can you please add a description for this, why it's needed and how it's supposed to work?